torch/fx/experimental/symbolic_shapes.py

```diff
@@ -255,29 +262,33 @@ def _nyi():
     'lt': lambda a, b: sympy.Lt(a, b),
     'le': lambda a, b: sympy.Le(a, b),
     'ge': lambda a, b: sympy.Ge(a, b),
+    'floor': lambda a: sympy.floor(a),
     'sym_float': lambda a: a,  # TODO: why can't I wrap with sympy.Float?
-    'sym_int': lambda a: _nyi(),
+    'sym_int': lambda a: sympy.floor(a),
```

Comment:
This is not correct.

```
>>> int(2.5)
2
>>> int(-2.5)
-2
>>> import math
>>> math.floor(-2.5)
-3
```

Reply:
No, I cannot use sympy.Interger(a) here, coz a could be a SymFloat, and sympy.Interger only accept python numerics

Reply:
I tried to compose sym_int using `sympy.floor(a) if a > 0 else sympy.ceiling(a)`
but I am not sure if a > 0 is created as a guard

Reply:
To guard on a being positive or negative is OK, but I think it is necessary to make the guard get recorded somehow, because otherwise we will record the wrong sympy expression for the other case.

Maybe the easiest thing to do is to STOP sym_int from being a node in the graph, and just have it be a Python function like:

```
def sym_int(a):
    if a > 0:
        return math.floor(a)
    else:
        return math.ceil(a)
```

seems pretty reasonable and very simple.